libswirl/hw/pvr/pvr_mem.cpp

```diff
@@ -103,7 +104,11 @@ void TAWrite(u32 address,u32* data,u32 count, u8* vram)
 	u32 address_w=address&0x1FFFFFF;//correct ?
 	if (address_w<0x800000)//TA poly
 	{
+		#if FEAT_TA == TA_HLE
```

Comment:
Compile time toggle between backends - #define for zero perf cost for builds like emscripten